kernel/kernel-api/src/test/scala/io/delta/kernel/internal/commit/CommitMetadataSuite.scala

```diff
@@ -38,151 +38,100 @@ class CommitMetadataSuite extends AnyFunSuite
 
   test("constructor validates non-negative version") {
     val ex = intercept[IllegalArgumentException] {
-      new CommitMetadata(
-        -1L, // negative version
```

Comment:
Is this `negative` comment relevant? maybe expand the comment?

Reply:
Before, with just `new CommitMetadata(-1L <comment>, ...)` -- there was no named param, and that comment was very helpful.

Now, with `createCommitMetadata(version = -1L)`, coupled with the test name `constructor validates non-negative version"` --> IMO things are sufficiently clear 👍